web/src/beta/features/PluginPlayground/Plugins/index.tsx

```
@@ -44,89 +44,105 @@
-            {category.plugins.map((plugin) => (
+      <IconList>
+        <Icon icon="addFile" size={32} />
+        <Icon icon="import" size={32} />
+        <Icon icon="export" size={32} />
+        <HorizontalSpacing />
+        <Icon icon="paperPlaneTilt" size={"normal"} />
+      </IconList>
+      <PluginListWrapper>
+        <PluginList>
+          {presetPlugins.map((category) => (
+            <div>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)